src/compass/utils/lut.py

```diff
@@ -183,8 +230,30 @@ def compute_geocoding_correction_luts(burst, dem_path,
     rg_set = resize(rg_set_temp, out_shape, **kwargs)
     az_set = resize(az_set_temp, out_shape, **kwargs)
 
+    # Compute wet and dry troposphere delays using RAiDER
+    wet_los_tropo, dry_los_tropo = np.zeros(out_shape), np.zeros(out_shape)
```

Comment:
```suggestion
    wet_los_tropo = np.zeros(out_shape)
    dry_los_tropo = np.zeros(out_shape)
```
Nit - one line per assignment

Or with less repetition and single line
```suggestion
    wet_los_tropo, dry_los_tropo = [np.zeros(out_shape) for _ in range(2)]
```